central_v2/src/pages/Login.tsx

```diff
@@ -1,143 +1,219 @@
+        </UpperLogin>
+        <OrText>or</OrText>
+        <MiddleContainer>
+          <UserTextField
+                variant="outlined"
+                placeholder="Username or Email"
+                value={userName}
+                onChange={(event) => setUserName(event.target.value)}
           />
-          <GoogleLogin
-            onSuccess={(googleToken) => {
-              handleGoogleLogin(googleToken.credential ?? '');
-            }}
-            onError={() => {
-              setAdminError(true);
-            }}
+          <UserTextField
+                variant="outlined"
+                placeholder="Password"
+                value={password}
+                onChange={(event) => setPassword(event.target.value)}
           />
-        </form>
-      </Grid>
-      {adminError ? (
-        <ErrorType>
-          {' '}
-          There has been an error. Please verify your username/password and
-          contact the administrator for account verification.{' '}
-        </ErrorType>
-      ) : null}
-    </Grid>
+          <ForgotPasswordButton onClick={handleForgotPasswordClick}>
+            Forgot your password?
+          </ForgotPasswordButton>
+        </MiddleContainer>
+        <LoginContainer>
+          <CentralButton buttonType={buttonTypeLogin} isEnabled={isLoginEnabled} onClick={handleLoginClick}/>
+        </LoginContainer>
+        <SignupContainer>
+          <NoAccountText>Dont have an account?</NoAccountText>
+          <CentralButton buttonType={buttonTypeSignup} isEnabled={isSignupEnabled} onClick={handleSignupClick}/>
+        </SignupContainer>
+      </InnerBodyContainer>
+    </SignUpMainContainer>
   );
-};
+}
 
-export default Login;
+export default function LoginSwitch() {
+  const [isPasswordForgot, setisPasswordForgot] = useState(false); // Track submission state
 
-const Field = styled(TextField)({
-  margin: '10px 0',
-  borderRadius: '20px',
-  width: '100%',
-});
-
-const SignUpLink = styled(Link)({
-  backgroundColor: '#FC1047',
-  textDecoration: 'none',
-  color: 'white',
-  borderRadius: '34px',
-  minWidth: '70px',
-  textAlign: 'center',
-  padding: '1vw',
-  whiteSpace: 'nowrap',
-  fontWeight: 'bold',
-});
-
-const LogInLink = styled(Link)({
-  backgroundColor: '#159EFA',
-  textDecoration: 'none',
-  color: 'white',
-  borderRadius: '34px',
-  minWidth: '70px',
-  textAlign: 'center',
-  padding: '1vw',
-  whiteSpace: 'nowrap',
-  fontWeight: 'bold',
-});
-
-const ButtonGrid = styled(Grid)({
-  display: 'flex',
-  flexDirection: 'row',
-  justifyContent: 'center',
-  alignItems: 'flex-start',
-  width: '10vw',
-  marginBottom: '2vw',
-  marginTop: '2vw',
-  gap: '10%',
-});
-
-const ErrorType = styled(Typography)({
-  fontStyle: 'italic',
-  textAlign: 'center',
-  color: 'grey',
-  paddingLeft: '5vw',
-  paddingRight: '5vw',
-});
+  const handleForgotPasswordClick = () => {
+    setisPasswordForgot(true)
+  };
+
```

Comment:
I'd probably break Login into a separate file here, to keep it super clear that Login is one component and LoginSwitch is another